store/job_store.go

```diff
@@ -20,9 +23,35 @@ type DataStore struct {
 //JobsMap is a map used for storing Job resources with the keys being string values.
 var JobsMap = make(map[string]models.Job)
 
+//LastUpdatedSlice is a type that implements the sort interface so that the jobs in it can be sorted using the generic Sort function.
+type LastUpdatedSlice []models.Job
+
+//Len is a function that's required by the sort interface.
+func (s LastUpdatedSlice) Len() int {
+	return len(s)
+}
+
+//Less is a function that's required by the sort interface.
```

Comment:
Less and Swap functions are very nice; if it wasn't for moving to using a database in the near future I would ask for some unit tests for these methods but alas I would refrain from doing so.

Reply:
thanks : )